### PR TITLE
node-751-restriction-on-duration-date-new-project-assignment

### PR DESCRIPTION
**Pull request by chadiegilnerubia**
Ticket ID:
New Project Assignment - Add a restriction on employee's assignment duration date
[#751](https://github.com/nerubia/kh360-react/issues/751)
FE PR: https://github.com/nerubia/kh360-react/pull/780

### CODE REVIEW

**Review by gborgonia-ir — APPROVED** [review]

File: src/controllers/admin/project-member-controller.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ import * as ProjectMemberService from "../../services/project-member-service"
 import CustomError from "../../utils/custom-error"
 import { ValidationError } from "yup"
 import { createProjectMemberSchema } from "../../utils/validation/project-member-schema"
+import { getById } from "../../services/project-service"
```
Comment (emmaacha) [inline src/controllers/admin/project-member-controller.ts:6]: Import this like
`import * as ProjectService from "../../services/project-service"`

File: src/controllers/admin/project-member-controller.ts [hunk 2]
```diff
@@ -86,6 +87,21 @@ export const store = async (req: Request, res: Response) => {
       allocation_rate,
     })
 
+    const projectDetails = await getById(parseInt(project_id))
```
Comment (emmaacha) [inline src/controllers/admin/project-member-controller.ts:90]: Then please change this to `await ProjectService.getById(parseInt(project_id))`

File: src/controllers/admin/project-member-controller.ts [hunk 3]
```diff
@@ -160,6 +175,19 @@ export const update = async (req: Request, res: Response) => {
       end_date,
       allocation_rate,
     })
+    const projectDetails = await getById(parseInt(project_id))
```
Comment (emmaacha) [inline src/controllers/admin/project-member-controller.ts:178]: Same here

**Review by emmaacha — CHANGES_REQUESTED** [review]

File: src/controllers/admin/project-member-controller.ts (hunk 2 above)
Comment (chadiegilnerubia) [reply src/controllers/admin/project-member-controller.ts:90]: Okey noted :3

**Review by emmaacha — APPROVED** [review]

**Pull request merged**